nullaway/src/main/java/com/uber/nullaway/ErrorBuilder.java

```diff
@@ -92,7 +92,7 @@ public class ErrorBuilder {
    *     expression into a @NonNull target, and this parameter is the Symbol for that target.
    * @return the error description
    */
-  Description createErrorDescription(
+  public Description createErrorDescription(
```

Comment:
This is because the PR moves the generics code into its own new package, so it cannot see this method anymore (default visibility for Java methods is "package private").